multichain-aggregator/multichain-aggregator-proto/proto/v1/multichain-aggregator.proto

```diff
@@ -139,7 +139,7 @@ message ListAddressesResponse {
 
 message ListTokensRequest {
   string q = 1;
-  optional string chain_id = 2;
+  repeated string chain_id = 2;
```

Comment:
```suggestion
  repeated string chain_ids = 2;
```
or is it too late to rename?

Reply:
It would require additional PR in contracts info service and a corresponding version release. I guess we can do it later for consistency